x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/flyout/steps/details/unfreeze_details_step.tsx

```diff
@@ -151,33 +152,44 @@ export const UnfreezeDetailsFlyoutStep: React.FunctionComponent<{
                   </EuiText>
                 ),
               },
+              /* We cannot unfreeze backing indices (that would break the related data_stream) */
```

Comment:
```suggestion
              /* We cannot reindex backing indices in the same way as regular indices (that would break the related data_stream) */
```